connectors/vault/vault_connector.go

```diff
@@ -42,17 +43,17 @@ func (s *server) GetCredentialsInfo(ctx context.Context, in *pb.DatasetCredentia
 }
 
 func main() {
-	vaultAddress := getEnv(VaultAddressKey)
-	timeOutInSecs := getEnvWithDefault(VaultTimeoutKey, defaultTimeout)
-	timeOutSecs, err := strconv.Atoi(timeOutInSecs)
-	port := getEnvWithDefault(VaultConnectorPortKey, defaultPort)
+	vaultAddress := vltutils.GetEnv(vltutils.VaultAddressKey)
+	timeOutInSecs := vltutils.GetEnvWithDefault(vltutils.VaultTimeoutKey, vltutils.DefaultTimeout)
```

Comment:
Neither are really "vault" utilities. Look for a package that has similar or create such functions in `pkg` and reuse (e.g., in `pkg/envvars`).